tests/Makefile

```diff
@@ -63,14 +63,15 @@ BINARIES := $(addsuffix $(EXEXT),$(APPS))
 
 .SILENT:
 
-.PHONY: all check test clean
+.PHONY: all c_files check test clean
```

Comment:
I don't see the change of Python version like in 4393.

Reply:
I botched my rebase and missed the most important commit... I'm also going to tweak the changelog entry and commit messages, because CMake in 2.16 may invoke Python 2 or 3 depending on the system. The point of the change is to stop requiring `python2`.